events/templates/event.html

```diff
@@ -44,10 +44,10 @@ <h6 class="float-right inline"> posted by:
     <p class="card-text container border-gray border">{{event.details}}</p>
     <!-- Users Signed Up -->
     {% with signed_up=event.signed_up.all %}
-      <p>Number signed up: {{ signed_up.count }}</p>
+    <!--  <p>Number signed up: {{ signed_up.count }}</p> -->
 
         <!-- Button trigger modal -->
-        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#modal_button{{event.id}}">
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#modal_button{{event.id}}" >
           Users Attending
```

Comment:
Add the number of users here so we can see in the event list view

Reply:
So not when the button is clicked?

Reply:
Should we have the number of users shown:

Below the users attending button?
When the button is clicked?
or as the actual text of the users attending button?

Because if I insert it here then the users attending button will have the number of users text alongside it.
Sorry for my confusion haha

Reply:
As the button text. It should read "### users attending".
In the modal is fine too (in addition to the button).

Reply:
As the button text. It should read "### users attending".
In the modal is fine too (in addition to the button).

Reply:
Ok thanks for the clarification